env/.env.clinic

```diff
@@ -1,3 +1,3 @@
-REACT_APP_USE_EEG="true"
-REACT_APP_USE_PHOTODIODE="true"
-REACT_APP_VOLUME="true"
\ No newline at end of file
+VITE_USE_EEG="true"
+VITE_USE_PHOTODIODE="true"
+VITE_VOLUME="true"
```

Comment:
Did you need to change those?

Reply:
I see, they're only used for the build

Reply:
Yeah, these are the "equipment" that users can turn on or off with environment variables